v2/observer/observer.go

```diff
@@ -110,6 +125,17 @@ func (o *observer) run(ctx context.Context) error {
 	// Wait for an error or a completed context
 	var err error
 	select {
+	// In essence, this comprises the Observer's "healthcheck" logic.
+	// Whenever we receive an error on this channel, we cancel the context and
+	// shut down.  E.g., if one loop fails, everything fails.
+	// This includes:
+	//   1. an error pinging the API server endpoint
+	//      (Observer <-> API comms)
+	//   2. TODO
+	//      (Observer <-> K8s comms)
```

Comment:
Out of scope for this PR or no?

Reply:
Thanks for the reminder; healthcheck added.